### PR TITLE
Update Ember release instructions to cover tool-new-release

### PR DESCRIPTION
**Pull request by locks**
These changes complicate the structure of the repo slightly, but I wanted to both have a written record of what the manual steps are and keep them separate from the instructions on how to run the tool.
Please review and comment!

### CODE REVIEW

File: ember-releases/README.md [hunk 1]
```diff
@@ -0,0 +1,25 @@
...
+There are two ways to go about it.
+Consult the relevant document for more detailed instructions:
+
+1. [Use tool-new-release](./tool-new-release.md)
+
+This tool tries to automate the process as much as possible as should be the preferred method.
+
+2. [Follow steps manually](./manual.md)
+
+If you wish to do the process manually, you will find all the steps described here.
```
Comment (ijlee2) [inline ember-releases/README.md:25]: **suggestion**:

From lines 17-25, I didn't think it was immediately clear which sentence goes with which of the two options. How about:

```suggestion
Consult the relevant document for more detailed instructions:

1. [Use tool-new-release](./tool-new-release.md). The tool tries to automate the process as much as possible. This option is the preferred method.
2. [Make a release manually](./manual.md).
```

File: ember-releases/tool-new-release.md [hunk 2]
```diff
@@ -0,0 +1,19 @@
+# tool-new-release process
+
+At the moment the tool only handles minor version bumps and requires the user to specify the version, like so:
+
+```bash
+tool-new-release --version 3.24.0
+```
```
Comment (ijlee2) [inline ember-releases/tool-new-release.md:7]:
```suggestion
At the moment, the tool only handles minor version bumps and requires you to specify the version:

```bash
# Going from 3.23 (current) to 3.24 (new)
tool-new-release --version 3.24.0
```

File: ember-releases/tool-new-release.md [hunk 3]
```diff
@@ -0,0 +1,19 @@
...
+
+To start the release process using `tool-new-release`, do the following:
```
Comment (ijlee2) [inline ember-releases/tool-new-release.md:9]:
```suggestion
To start the release process using `tool-new-release`:
```

File: ember-releases/tool-new-release.md [hunk 4]
```diff
@@ -0,0 +1,19 @@
...
+
+1. Go to the [releases page](https://github.com/ember-learn/tool-new-release/releases)
+2. Find the most recent draft
```
Comment (ijlee2) [inline ember-releases/tool-new-release.md:12]: Initially, I wasn't sure what you meant by "draft" until I clicked on the link in step 1.

After the sentence for step 2, I think (1) highlighting the word "draft" in bold, and (2) attaching a screenshot would help a person who is new to making a release:

 ```suggestion
2. Find the most recent **draft**

    <img alt="description of screenshot" src="link to uploaded screenshot">
```

File: ember-releases/tool-new-release.md [hunk 5]
```diff
@@ -0,0 +1,19 @@
...
+3. Expand the "Assets" panel
```
Comment (ijlee2) [inline ember-releases/tool-new-release.md:13]:
```suggestion
3. Expand the **Assets** panel
```

File: ember-releases/tool-new-release.md [hunk 6]
```diff
@@ -0,0 +1,19 @@
...
+4. Download the binary that corresponds to your operating system
+5. Run the binary
+5.1. If you are on macOS you will need to right-click and "Open" the binary to get around the security check, since the binary is not signed
```
Comment (ijlee2) [inline ember-releases/tool-new-release.md:16]:
```suggestion
5. Run the binary. If you are on macOS, you will need to right-click and "Open" the binary to get around the security check (the binary is not signed).

    <img width="900" alt="On Mac, right-click on the binary file and click Open" src="https://user-images.githubusercontent.com/16869656/104086195-ed305e80-5255-11eb-97e1-00388e2ba85c.png">
```

File: ember-releases/tool-new-release.md [hunk 7]
```diff
@@ -0,0 +1,19 @@
...
+6. Follow the instructions presented carefully
```
Comment (ijlee2) [inline ember-releases/tool-new-release.md:17]:
```suggestion
6. Carefully follow the instructions presented in the terminal
```

File: ember-releases/tool-new-release.md [hunk 8]
```diff
@@ -0,0 +1,19 @@
...
+
+For more detailed information refer to the [README](https://github.com/ember-learn/tool-new-release).
```
Comment (ijlee2) [inline ember-releases/tool-new-release.md:19]:
```suggestion
For more information about tool-new-release, please see [README](https://github.com/ember-learn/tool-new-release).
```

**Review by ijlee2 — APPROVED** [review]
Thanks for documenting making a release for Ember Data and running `tool-new-release`! I made several suggestions for steps for `tool-new-release`. I think incorporating these suggestions will help.

File: ember-releases/tool-new-release.md (hunk 6 above)
Comment (locks) [reply ember-releases/tool-new-release.md:16]: Thanks for the image!

**Pull request closed without merging**